sr_grasp/src/sr_grasp/__init__.py

```diff
@@ -46,30 +45,26 @@ def __init__(self):
     def from_msg(cls, msg):
         """Construct a shadow grasp object from moveit grasp object."""
         grasp = Grasp()
-        grasp.id = msg.id
-        grasp.pre_grasp_posture = deepcopy(msg.pre_grasp_posture)
-        grasp.grasp_posture = deepcopy(msg.grasp_posture)
-        grasp.grasp_pose = deepcopy(msg.grasp_pose)
-        grasp.grasp_quality = msg.grasp_quality
-        grasp.pre_grasp_approach = deepcopy(msg.pre_grasp_approach)
-        grasp.post_grasp_retreat = deepcopy(msg.post_grasp_retreat)
-        grasp.post_place_retreat = deepcopy(msg.post_place_retreat)
-        grasp.max_contact_force = msg.max_contact_force
-        grasp.allowed_touch_objects = deepcopy(msg.allowed_touch_objects)
+        grasp.id = msg.id  # pylint: disable=C0103,W0201
+        grasp.pre_grasp_posture = deepcopy(msg.pre_grasp_posture)  # pylint: disable=W0201
+        grasp.grasp_posture = deepcopy(msg.grasp_posture)  # pylint: disable=W0201
+        grasp.grasp_pose = deepcopy(msg.grasp_pose)  # pylint: disable=W0201
+        grasp.grasp_quality = msg.grasp_quality  # pylint: disable=W0201
+        grasp.pre_grasp_approach = deepcopy(msg.pre_grasp_approach)  # pylint: disable=W0201
+        grasp.post_grasp_retreat = deepcopy(msg.post_grasp_retreat)  # pylint: disable=W0201
+        grasp.post_place_retreat = deepcopy(msg.post_place_retreat)  # pylint: disable=W0201
+        grasp.max_contact_force = msg.max_contact_force  # pylint: disable=W0201
+        grasp.allowed_touch_objects = deepcopy(msg.allowed_touch_objects)  # pylint: disable=W0201
```

Comment:
bit weird as Grasp should have those variables in the constructor defined after inheriting from moveit_msgs...

Reply:
Im not sure how this works as I don't work with msg's much, but we can have a talk with Beatriz about it tommorow as this aws one of her changes :)

Reply:
after a chat with @bdenoun we think it's mostly an pylinter issue, would be great to investigate this issue

Reply:
I think it likely is the case, maybe it doesn't support msg's